interp/notation.ml

```diff
@@ -2497,6 +2497,29 @@ let interp_notation_as_global_reference_expanded ?loc ~head test ntn sc =
 let interp_notation_as_global_reference ?loc ~head test ntn sc =
   let _,_,_,_,ref = interp_notation_as_global_reference_expanded ?loc ~head test ntn sc in ref
 
+let pr_id_infos (id, ((level,(tmp_scopes, scopes)), under_binders, kind)) =
+  let scopes = List.map (fun x -> "_"^x) tmp_scopes @ scopes in
+  match scopes with
+  | [] -> None
+  | _ ->
+    (* IDK how to insert delimiters in the constr for printing
+       so instead use "in constr" modifier even though it's ignored
+       (#20297)
+       Also not sure if we can ever see multiple scopes but if we do we produce invalid syntax.
```

Comment:
Seems to work well:
```coq
Arguments S _%_nat_scope%_bool_scope.

Notation "n .+1" := (S n).

Locate "_ .+1".
(* gives:
Notation "n .+1" := (S n) (* n in scopes _nat_scope _bool_scope *)
*)
```

Reply:
I mean `Notation "n .+1" := (S n) (n in scopes _nat_scope _bool_scope)` is not valid syntax (AFAIK)